README.md

```diff
@@ -109,30 +104,30 @@ accomplish this as follows:
         Note: See the GREASE-like discussion below for how we could anticipate the inevitable lies
         which user agents might want to tell in this field.
 
-    2.  The `Sec-CH-UA-Platform` header field represents the platform's brand and major version. For example:
+    2.  The `Sec-CH-Platform` header field represents the platform's brand and major version. For example:
```

Comment:
Is this a change we're making? I kinda liked that the `UA-` prefix grouped all these together (and some of them are probably too generically named without it).

Reply:
I thought so...
As I see it, we have two options:
Either we align all related FP and Accept-CH strings to say "ua-" as well, or we remove it here. But 3 prefixes is starting to be a lot...